Example/TABObserverSet/Helpers/UIAlertController+ObserverSet.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  UIAlertController+ObserverSet.swift
+//  TABObserverSet
+//
+//  Created by Luqman Fauzi on 20/08/2017.
+//  Copyright © 2017 The App Business. All rights reserved.
+//
+
+import UIKit
+import TABObserverSet
+
+extension UIAlertController {
+
+    typealias Grocery = (String, Int)
+
+    func presentGroceryInput<T: UIViewController>(_ target: T, observer: ObserverSet<Grocery>) {
```

Comment:
What's the benefit of specialising this type? If there's no immediate benefit I'd opt for just declaring the type in the argument directly to help minimise confusion for people who are newer to developing in Swift.